index.html

```diff
@@ -1,9 +1,96 @@
-NPM Clone Nifty Penguin Magic npm Enterprise Products Solutions Resources Docs Support Search Join Log In Build amazing
-things Essential JavaScript development tools that help you go to market faster and build powerful applications using
-modern open source code. See plans Join for free Bring the best of open source to your company npm is the tool used by
-over 11,000,000 JavaScript developers around the world. Your developers already use it. Your company depends on it.
-Create an Org and get more out of the tools your team already knows and loves. Zero configuration Create an org, add
-your team, and start collaborating. Nothing to configure, set up, or manage. Team management Control who has access to
-what modules within your team namespace using straightforward team management capabilities. Familiar features npm Orgs
-has 100% parity with all the public npm registry features your developers already use. npm audit Enjoy the security
-auditing features built into the npm client, a zero-friction way to make open source software safer. Create an Org
+<title>npm Enterprise</title>
+<head>
+    <link rel="stylesheet" type="text/css" href="npm.css" />
+</head>
+
+
+<header>
+    <div>
+      <div>
+        <img class="blackHeart" src="images/black-heart.png" alt="Black Heart" />
+        <span> Nifty Penguin Magic </span>
+      </div>
+      <nav>
+        <ul>
+          <li><a href="#"> npm Enterprise </a></li>
+          <li><a href="#"> Products </a></li>
+          <li><a href="#"> Solutions </a></li>
+          <li><a href="#"> Resources </a></li>
+          <li><a href="#"> Docs </a></li>
+          <li><a href="#"> Support </a></li>
+        </ul>
+      </nav>
+    </div>
+  
+    <searchBar >
+
+        <ul>
+            <li><img class="logoSize" src="images/npm-logo.png" alt="NPM" /> </li>
+            <li><img class="glassSize" src="images/magnifying-glass.png " alt="Search" /> </li>
+            <li><form id="form">
+              
+              <input type="text"  placeholder="Search Packages" size="100">
+              <input id="submit" type="submit" value="Search">
+              
+              </form></li> <! -- form here -->
+            
+            <li class="searchBarButton"><a href="#"> Join </a></li>
+            <li class="searchBarButton"><a href="#"> Log In </a></li>
+          
+          </ul>
+    <!-- Search Join Log In  -- add logo, search bar and buttons (or links and style them as buttons later) -->
+    </searchBar>
+      
+  </header>
+
+<section id="firstSection">
+    <h1>Build amazing things</h1>
+
+    <article class="article">Essential JavaScript development tools that help you go to market faster and build powerful applications using
+    modern open source code.</article>
+    <div class="twoButtons">
+      <div class="linkHighlight">
+        See plans
+      </div> <div class="linkDontHighlight">
+        Join for free
+      </div>
```

Comment:
The two divs inside <div class="twoButtons"> should be two links styled as buttons.